NAMESPACE

```diff
@@ -8,6 +8,14 @@ S3method(as_glue,character)
 S3method(as_glue,default)
 S3method(as_glue,glue)
 S3method(print,glue)
+S3method(testthat::compare,glue)
+S3method(vctrs::vec_cast,character.glue)
+S3method(vctrs::vec_cast,glue.character)
+S3method(vctrs::vec_cast,glue.glue)
+S3method(vctrs::vec_ptype2,character.glue)
+S3method(vctrs::vec_ptype2,glue.character)
+S3method(vctrs::vec_ptype2,glue.glue)
+S3method(waldo::compare_proxy,glue)
```

Comment:
glue doesn't (and cannot) depend on testthat, vctrs, waldo, etc. It feels like we're dismantling the conditional and delayed export of these methods and are, instead, assuming that these packages are available.

Am I out of the loop and this is actually OK now?

Reply:
That's exactly the point of this form of `S3method()` 😄

Reply:
Upon further research, this is roxygen2 newly taking advantage of a feature that appeared in R 3.6:

https://rstudio.github.io/r-manuals/r-exts/Creating-R-packages.html#registering-s3-methods

> As from R 3.6.0 one can also use `S3method()` directives to perform delayed registration. With
>
> ``` r
> if(getRversion() >= "3.6.0") {
>     S3method(pkg::gen, cls)
> }
> ```
>
> function `gen.cls` will get registered as an S3 method for class `cls` and generic `gen` from package pkg only when the namespace of pkg is loaded. This can be employed to deal with situations where the method is not “immediately” needed, and having to pre-load the namespace of pkg (and all its strong dependencies) in order to perform immediate registration is considered too onerous.

Reply:
Ok, really old then 😆